shell/platform/android/test/io/flutter/embedding/engine/FlutterEngineGroupComponentTest.java

```diff
@@ -232,11 +259,72 @@ public void canCreateAndRunWithCustomEntrypointArgs() {
                 .setDartEntrypointArgs(secondDartEntrypointArgs));
 
     assertEquals(2, engineGroupUnderTest.activeEngines.size());
-    verify(mockflutterJNI, times(1))
+    verify(mockFlutterJNI, times(1))
         .spawn(
             nullable(String.class),
             nullable(String.class),
             nullable(String.class),
             eq(secondDartEntrypointArgs));
   }
+
+  @Test
+  public void createEngineSupportMoreParams() {
```

Comment:
We also need to check in this test that `automaticallyRegisterPlugins` is working, it seems a little difficult though.